OpenRA.Mods.Common/Traits/Radar/JamsRadar.cs

```diff
@@ -18,6 +18,9 @@ public class JamsRadarInfo : TraitInfo<JamsRadar>
 	{
 		[Desc("Range for jamming.")]
 		public readonly WDist Range = WDist.Zero;
+
+		[Desc("What diplomatic stances are affected.")]
+		public readonly Stance JamsRadarStances = Stance.Ally | Stance.Neutral | Stance.Enemy;
```

Comment:
Please make this default to Neutral | Enemy or add an upgrade rule because this regresses third-party mods as-is.